src/Common/src/System/Dynamic/Utils/DelegateHelpers.cs

```diff
@@ -10,6 +10,10 @@
 using System.Runtime.CompilerServices;
 using System.Reflection;
 
+#if FEATURE_REFEMIT
```

Comment:
I may need another flag to switch between compiling and interpreting flavors. I just want to get the sources to be the same as a first step.